generators/client/templates/vue/package.json.ejs

```diff
@@ -33,6 +33,9 @@ limitations under the License.
     "axios": "0.18.0",
     "bootstrap": "4.1.3",
     "bootstrap-vue": "2.0.0-rc.11",
+    <%_ if (clientTheme && clientTheme !== 'none') { _%>
```

Comment:
to avoid checking if clientTheme is not undefined (everywhere) , you can init it at the start, if the value is not defined

Reply:
Changed it.